.github/workflows/ci.yaml

```diff
@@ -6,6 +6,68 @@ on:
     - cron: "0 */2 * * *"
```

Comment:
no need for cron

Reply:
Good catch. Do you mean that I should remove in this PR? I think so since it may fail due to the versions not being different.

Reply:
yeah rm in this PR

Reply:
Left in bc it may be useful for OSS releases that are not in sync with agent released. E.g. if OSS releases a change that breaks the agent, this scheduled test will fail and will run even if we aren't making changes to the agent. I did make a change to only run the version bump jobs if we are in a PR though (and not on schedule) so that only the CI tests run on the schedule.

Reply:
ohh this is for all of CI. Sorry I thought you had added it. yeah makes sense to leave

Reply:
All good - great catch. The version bump check would have failed on scheduled runs.